ui/src/components/settings/sub_pages/general.rs

```diff
@@ -36,9 +36,11 @@ pub fn GeneralSettings(cx: Scope) -> Element {
                 section_label: get_local_text("settings-general.overlay"),
                 section_description: get_local_text("settings-general.overlay-description"),
                 Switch {
-                    active: state.read().configuration.general.enable_overlay,
+                    // TODO: This overlay causes a crash in windows
+                    disabled: if cfg!(target_os = "windows") {true } else {false},
```

Comment:
It will only return false if the target isnt windows

```suggestion
                    disabled: cfg!(target_os = "windows"),
```